modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
```

Comment:
Take a look at all the Labs listed in this topic--not sure I see how they are related or if they use leader pinning. If that is the case, you can remove them by omitting the categories tag here.

Reply:
@Feediver1 they're not really -- Follower fetching also has these page categories. I think these are just the labs available for the two as they are pretty broad. We don't have any for multi-AZ deployments which would be the most relevant topic. Probably better to remove.